### PR TITLE
MultipartKit proposal

### PR DESCRIPTION
**Pull request by siemensikkema**
Add MultipartKit to the sandbox packages

### CODE REVIEW

File: proposals/NNNN-multipart-kit.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
+# MultipartKit
+
+* Proposal: [SSWG-NNNN](NNNN-multipart-kit.md)
+* Authors: [Siemen Sikkema](https://github.com/siemensikkema), [Tim Condon](https://github.com/0xTim)
+* Review Manager: TBD
+* Status: **Awaiting implementation**
+* Implementation: [repo name](https://github.com/vapor/multipart-kit)
```
Comment (adam-fowler) [inline proposals/NNNN-multipart-kit.md]: Shouldn't `repo name` be MultipartKit

File: proposals/NNNN-multipart-kit.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+* Forum Threads: [Pitch](https://forums.swift.org/t/multipartkit), [Discussion](https://forums.swift.org/), [Review](https://forums.swift.org/)
```
Comment (adam-fowler) [inline proposals/NNNN-multipart-kit.md]: Given there are no discussion or review thread, we can remove these. I guess a proposal thread will be added when it goes up.

File: proposals/NNNN-multipart-kit.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+## Package Description
+
+Multipart parser and serializer with `Codable` support for Multipart Form Data.
+
+|  |  |
+|--|--|
+| **Package Name** | `multipart-kit` |
+| **Module Name** | `MultipartKit` |
+| **Proposed Maturity Level** | [Sandbox](https://github.com/swift-server/sswg/blob/main/process/incubation.md#process-diagram) |
+| **License** | [TBD](https://choosealicense.com/licenses/mit/) |
```
Comment (adam-fowler) [inline proposals/NNNN-multipart-kit.md]: TBD should be MIT

**Review by adam-fowler — COMMENTED** [review]
Looking good, some minor edits
I'm going to test this with Hummingbird to see how it works outside the Vapor eco-system.

**Comment by siemensikkema** [comment]
@adam-fowler Thanks for comments and for trying it out! Curious to learn about your findings!

File: proposals/NNNN-multipart-kit.md [hunk 4]
```diff
@@ -0,0 +1,108 @@
+# MultipartKit
+
+* Proposal: [SSWG-NNNN](NNNN-multipart-kit.md)
+* Authors: [Siemen Sikkema](https://github.com/siemensikkema), [Tim Condon](https://github.com/0xTim)
+* Review Manager: TBD
```
Comment (kmahar) [inline proposals/NNNN-multipart-kit.md]: I'll be the review manager for this one.
```suggestion
* Review Manager: [Kaitlin Mahar](https://github.com/kmahar)
```

File: proposals/NNNN-multipart-kit.md [hunk 5]
```diff
@@ -0,0 +1,108 @@
+# MultipartKit
+
+* Proposal: [SSWG-NNNN](NNNN-multipart-kit.md)
```
Comment (kmahar) [inline proposals/NNNN-multipart-kit.md]:
```suggestion
* Proposal: [SSWG-0017](0017-multipart-kit.md)
```
(we should rename the file to this as well)

File: proposals/NNNN-multipart-kit.md [hunk 6]
```diff
@@ -0,0 +1,108 @@
...
+* Authors: [Siemen Sikkema](https://github.com/siemensikkema), [Tim Condon](https://github.com/0xTim)
+* Review Manager: TBD
+* Status: **Awaiting implementation**
```
Comment (kmahar) [inline proposals/NNNN-multipart-kit.md]:
```suggestion
* Status: **Implemented**
```

File: proposals/NNNN-multipart-kit.md [hunk 7]
```diff
@@ -0,0 +1,108 @@
...
+* Implementation: [MultipartKit](https://github.com/vapor/multipart-kit)
+* Forum Threads: [Pitch](https://forums.swift.org/t/multipartkit)
```
Comment (kmahar) [inline proposals/NNNN-multipart-kit.md]: (once this is merged in I will create a forums thread, and then update the link here)

**Review by kmahar — COMMENTED** [review]
proposal looks good and complete, just some small housekeeping comments

**Review by 0xTim — APPROVED** [review]
🎉

**Review by adam-fowler — APPROVED** [review]

**Comment by adam-fowler** [comment]
@siemensikkema By the way I integrated it into a hummingbird example with relative ease https://github.com/hummingbird-project/hummingbird-examples/tree/main/multipart-form

**Review by kmahar — APPROVED** [review]
LGTM, I'll create a forums thread to start gathering community feedback today

**Pull request merged**